.github/workflows/colcon_test_build.yaml

```diff
@@ -3,20 +3,21 @@ name: ROS2 test build
 on:
   push:
     branches:
-      - 'ros2'
+      - 'ros2-workflow-devel'
```

Comment:
Is future work planned on this branch and we don't want to switch to `ros2` yet?

Reply:
Good catch. I knew I will forget abut it...